examples/dynamic.c

```diff
@@ -31,6 +33,10 @@
 #include <unistd.h>
 #include <time.h>
 #include <sys/param.h>
+#include <pmix_config.h>
+#ifdef HAVE_NETDB_H
+#include <netdb.h>
+#endif
```

Comment:
Can you pinpoint what in this example required netdb.h? I ask because this is supposed to be an example user program, and including pmix_config.h would not generally be acceptable. Instead, we should remove from the example whatever is forcing that include file to be required.